tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -52,7 +52,7 @@ def test_scale_osds_reboot_nodes(self, interface, project_factory, multi_pvc_fac
         current_osd_count = count_cluster_osd()
         proj_obj = project_factory()
         if current_osd_count == 3:
-            while not validate_osd_utilization(osd_used=50):
+            while not validate_osd_utilization(osd_used=10):
```

Comment:
@Shrivaibavi Can you review this.
Filling up cluster to 50% and further adding OSD's to the cluster is taking huge amount of time, TC will take more than 1 day to complete, so Tiffany updated the testcase based on the bz requirement, please provide your opinion about this change.

Reply:
@ramkiperiy @Shrivaibavi 
With workers of m5.4xlarge and osd size of 2048, it takes couple days to fill up to 10%.  Filling up 50% will take more than 5 days in my observation. Therefore, we can reduce the % to make it runs a lot quicker since related bz doesn't require how much capacity of cluster used.

Reply:
@tiffany Sounds good :-)